### ISSUE TITLE
Zentyal 6.2.0 : DHCP marked as stopped but really is running

### ISSUE THREAD

**Issue by mado987** [zentyal/zentyal#1971]
**Describe the bug**

At dashboard, DHCP service is marked as stopped but really it's running.
Same issue as #1820  "Zentyal 5.1 : DHCP marked as stopped but really is running"

Solution is also the same:    systemctl restart tftpd-hpa.service

**To Reproduce**

No idea ....  this seems to happen randomly after a certain period of operation

**Zentyal OS (please complete the following information):**

 - Version: 6.2.0
 - OS: Ubuntu 18.04 LTS

**Comment by djoven89** [comment]
Hi @mado987 ,

We have tried to reproduce it but we couldn't. 

Did you analyze the log file '**/var/log/syslog**' ? Concretely, you should take a look at the following records:

* isc-dhcp-server
* tftp

Best regards, Daniel Joven.

**Comment by Neustradamus** [comment]
@mado987: Any news?

**Comment by mado987** [comment]
sorry for my silence ... I was fully busy with a lot of other stuff and I confess that I have since restarted with a fresh installation and changes in my network configuration.
This concern doesn't exist anymore, I have suspected a  2th dhcp server in my network or other network misconfigurations ...
It's indeed difficult to reproduce because that's doesn't arise imediatly afther a restart. 
Let's see if my fresh installation do the job (and this is the case until now).
I work with VMs under proxmox so if I catch some freetime in august I will test again from backup images.

Regards,
Dominique
